app/views/projects/_form.html.erb

```diff
@@ -1,4 +1,14 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
+
+<% if policy(@project).admin? && @project.project_access_type == 'Public' %>
```

Comment:
Can we combine this policy to `can_feature?` or similar

Reply:
Done :heavy_check_mark: